### ISSUE TITLE
Sign in to confirm you're not a bot

### ISSUE THREAD

**Issue by ilchoTaleski** [LonelyCpp/react-native-youtube-iframe#357]
**Describe the bug**
I am using [react-native-youtube-iframe](https://www.npmjs.com/package/react-native-youtube-iframe) in my react native mobile app and up until now I had no issues embedding youtube videos. We had iOS app released at first and just two months ago we released the android app, as well. Now, one user complained about not being able to watch the videos on the app. They got the message: Sign in to confirm you are not a bot.

Here is part of my code using the library:

```
import YoutubePlayer from 'react-native-youtube-iframe';

...

<YoutubePlayer
    height={hProp || videoHeight}
    width={wProp || videoWidth}
    play={playing}
    videoId={videoYoutubeId}
    onChangeState={handleStateChange}
    ref={playerRef}
    onReady={handleReady}
    initialPlayerParams={{
        rel: false,
    }}
/>
```

This is how I am extracting the videoId from the youtube link (the youtube links are fetched from my database)

```
const videoYoutubeId = getSearchParamFromURL(
    video.youtubeLink,
    'v'
);
```

And this is the function:

```
export const getSearchParamFromURL = (url, param) => {
    const include = url.includes(param);

    if (!include) return null;

    const params = url.split(/([&,?,=])/);
    const index = params.indexOf(param);
    const value = params[index + 2];
    return value;
};
```

**To Reproduce**
I cannot reproduce the behaviour. Based on user report, they cannot access any of the videos that are on the app. It is working for iOS devices and currently we have only one user that reported this issue on their android device.

**Screenshots**
This is what they sent to us:
![462549510_1605449763342599_8145747988130672576_n](https://github.com/user-attachments/assets/a0272110-13ea-43e2-a00d-212172b66c6a)

**Smartphone:**
 - Device: Galaxy A54 5G
 - OS + version: Android 13
 - `react-native-youtube-iframe` version: ^2.3.0
 - `react-native-webview` version: ^13.3.1

I would really need to fix this issue, since users are subscribed and paying to watch videos on the app.

**Comment by evgeny-ykm** [comment]
Same here

**Comment by paulsizer** [comment]
Anymore info here? Any workarounds?

**Comment by lutfi-haslab** [comment]
same here

**Comment by Riyaancode** [comment]
I'm also facing this issue on android

**Comment by ramses44-17** [comment]
I meet this issues on Firefox browser in my react web application

**Comment by sinankarakok** [comment]
Hello, this error message ("Sign in to confirm you're not a bot") tends to occur more frequently when a VPN is enabled.

**Comment by K-E-L** [comment]
I agree with @sinankarakok . It is definitely a networking problem coming from YouTube and has nothing to do with the YouTube client itself. Seems to be VPN related.
